src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy

```diff
@@ -585,8 +585,7 @@ class LeVADocumentUseCase extends DocGenUseCase {
                     name: r.name,
                     description: convertImages(r.description),
                     proposedMeasures: "Mitigations: ${ mitigationsText }<br/>Tests: ${ testsText }",
-                    requirements: requirements.findAll { it != null }.collect { it.name }.join("<br/>"),
-                    requirementsKey: requirements.findAll { it != null }.collect { it.key }.join("<br/>"),
+                    requirements: requirements.findAll { it != null },
```

Comment:
As there will only be one requirement, It would be better to call the entry and the variable requirement instead of requirements. This will affect the template. The method can also be called getRequirement.